sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Release History
 
+## 1.1.1 (Unreleased)
+
+* Update OpenTelemetry dependencies. This includes a hotfix for [GHSA-vh2m-22xx-q94f](https://github.com/open-telemetry/opentelemetry-dotnet/security/advisories/GHSA-vh2m-22xx-q94f).
```

Comment:
please include the change in the changelog itself esp. since it is a breaking change.

Reply:
also, mention how to get back the old behavior in the changelog itself.